src/lib/ansen-gpt/utils/pineconeClient.ts

```diff
@@ -0,0 +1,23 @@
+import { PineconeClient } from "@pinecone-database/pinecone";
+
+if (!process.env.PINECONE_ENVIRONMENT || !process.env.PINECONE_API_KEY) {
+  throw new Error("Pinecone environment or api key vars missing");
+}
+
+async function initPinecone() {
+  try {
+    const pinecone = new PineconeClient();
+
+    await pinecone.init({
+      apiKey: process.env.PINECONE_API_KEY ?? "",
```

Comment:
We might want "PINECONE_API_KEY" & "PINECONE_ENVIRONMENT" to be a const with a type casted "as string", as after the line 3 if process.env.PINECONE_API_KEY & process.env.PINECONE_ENVIRONMENT will not be undefined or empty

Suggestion:
```ts
const PINECONE_API_KEY = process.env.PINECONE_API_KEY
const PINECONE_ENVIRONMENT = process.env.PINECONE_ENVIRONMENT

if (!PINECONE_API_KEY || !PINECONE_ENVIRONMENT) {
   throw new Error("Pinecone environment or api key vars missing");
}

// Here both PINECONE_API_KEY and PINECONE_ENVIRONMENT will be a string with no need of type casting or "??" 
```